components/carousel.js

```diff
@@ -0,0 +1,154 @@
+  width: '95%', 
+  maxWidth: '6000px', 
+  height: '600px', 
+  padding: '40px', 
+  backgroundColor: '#f9f9f9', 
+  borderRadius: '10px', 
+  boxShadow: '0px 0px 10px rgba(0, 0, 0, 0.1)' ,  
+  textAlign: 'center',
+  '@media screen and (max-width:759px)': {
+  width: 'auto',
+  height:'2000px'
+},
+},
+image:{
+  height: '150px',
+  width: '150px', 
+  borderRadius: '50%',
+  '@media screen and (max-width:759px)': {
+  height: 'auto',
+  width: '100%',
+  maxWidth: '130px'
+  },
+},
+textParagraph:{
+  whiteSpace: 'pre-wrap', 
+  fontSize: '1.5rem',
+  fontWeight: '300px',
+  color:'#000',
+  marginBottom: '10px', 
+  fontFamily: ' "eurostile" , sans-serif',
+  '@media screen and (max-width:759px)': {
+  fontSize: '0.5rem',
+  textOverflow: 'ellipsis'
+  },
+},
+authorParagraphOne:{
+  color: '#141C3A', 
+  fontWeight: 'bold',
+  fontSize: '25px', 
+  '@media screen and (max-width:560px)':{
+    fontSize: '15px',
+  },
+},
+authorParagraphTwo:{
+  fontSize: '20px',
+  color: '#141C3A',
+  fontFamily:  '"europa ", sans-serif',
+  marginTop: '-1.25rem',
+  '@media screen and (max-width:560px)':{
+    fontSize: '10px',
+  },
+},
+radiocontainer:{
+  textAlign: 'center', 
+  transition: 'backgroundColor 0.25s',
+},
+radio1:{
+  display:'inline-block ', 
+  margin: '0 6px', 
+  width: '15px', 
+  height: '15px',
+  border: '1px solid #E6ECF8', 
+  borderRadius: '30%', 
+  cursor: 'pointer',
+},
+radio2:{
+  display:'inline-block ', 
+  margin: '0 6px', 
+  width: '15px', 
+  height: '15px',
+  border: '1px solid #E6ECF8', 
+  borderRadius: '30%', 
+  cursor: 'pointer',
+},
+}))
+
+
+const Carousel = () => {
+  const classes = useStyles();
+  const [activeIndex, setActiveIndex] = useState(0);
+  const testimonials = [
+    {
+        image: pascal1,
+        text:
+        '"Matt was a real pleasure to work with and we look \n forward to working with him again. He’s definitely \n the kind of designer you can trust with a project \n from start to finish."',
+        author1: "Pascal Tremblay",
+        author2: "Creative Lead, Good Kind",     
+    },
+    {
+        image: pascal2,
+        text:
+          '"Matts a clear communicator with the tenacity and \n confidence to really dig in to tricky design scenarios \n and the collaborative friction thats needed to \n produce excellent work."',
+          author1: "Alvin Engler",
+          author2: "CEO, West Third Enterprises, Inc.",       
+      },
+  ];
+
+  useEffect(() => {
+    const intervalId = setInterval(() => {
+      setActiveIndex((activeIndex + 1) % testimonials.length);
+      }, 5000);
+
+      return () => clearInterval(intervalId);
+    }, [activeIndex, testimonials]);
+    const handlePrevClick = () => {
+      setActiveIndex(
+      activeIndex === 0 ? testimonials.length - 1 : activeIndex - 1
+      );
+    };
+    const handleNextClick = () => {
+      setActiveIndex((activeIndex + 1) % testimonials.length);
+    };
+
+  return (
+    /******Whole Container Style ******/
```

Comment:
the whole return component should be in proper format ,line 134 to rest

Reply:
Not getting this point . Can you please drop any sample for reference . 

Line 134 is single comment line

Reply:
<div>
             <div>
                        <div> alignment should be like this, it will look good and also 
                                       easy to understand

Reply:
check slack plz